### ISSUE TITLE
Printing "long_name" class

### ISSUE THREAD

**Issue by ferball** [jekkos/opensourcepos#528]
Admittedly not much of a programmer, more of a hack but I get by...  
I ran into to problem of items not printing on the receipt I removed the "class=long_name" from the span tag and the problem was solved.  Not sure if this breaks something else or if I was doing something wrong in the first place but I thought I would throw it out there and somebody can set me straight.  I was not able to find the long_name in any of the CSS files, was I missing something or have I got things realy screwed up?

**Comment by daN4cat** [comment]
.long_name style class is defined in header_excel.php

However using it in receipt, invoice and invoice_email doesn't seem to make sense as there is no connection between the two.

I would need to check why is in header_excel, but it might be a miss of some past clean up.

**Comment by daN4cat** [comment]
And thanks for pointing this out.

**Comment by odiea** [comment]
The last version I found that in was dated 10/20/2015. He might consider an update to the newer version.

**Comment by daN4cat** [comment]
Indeed, it should not affect the printing in 3.0. So it must be an issue with something old.

However I need to clean up those style classes.

**Comment by daN4cat** [comment]
OK, I've done a bit of tidy up here thanks to this issue:

- excel header and footer are not required anymore as I cannot find any reference in bootstrap-table branch and therefore removed the partial files
- removed all the long_name and short_name style classes as there was even a .css clash between ospos_print.css and receipt.css files. short_name class was always switched off

Of course I regenerated the dist files.

I tested receipt and invoices and they seem fine, so this is done. If somebody can test printing of receipts and invoices, and emailing of the two we can then close if no issue. (NOTE: "ugly invoice/invoice email" is a different matter!)
